README.md

```diff
@@ -10,6 +10,10 @@ read responses.
 
     go get -v -u github.com/VoltDB/voltdb-client-go/voltdbclient
 
+# Requirements
+
+    go version >= 1.8
```

Comment:
i've seen that you'r using 1.8 and higher and thought that ur client is using >=1.8 version.
So we need to remove Port method and use old version methods.